tests/builtins/test_enumerate.py

```diff
@@ -2,7 +2,55 @@
 
 
 class EnumerateTests(TranspileTestCase):
-    pass
+    def test_enumerate(self):
```

Comment:
It looks like there's now a test for missing `items` arg. Is there one for empty list/iterable?